Jenkinsfile

```diff
@@ -181,7 +180,12 @@ stage('unit-tests') {
 
 stage('publish') {
   node('docker') {
-    publishReport('linux-sync')
-    publishReport('macOS-sync')
+    // we need sources to allow the coverage report to display them
+    rlmCheckout(scm)
+    // coverage reports assume sources are in the parent directory
+    dir("build") {
+      publishCoverageReport('linux-sync')
+      publishCoverageReport('macOS-sync')
```

Comment:
Unfortunately the cobertura plugin we use does not yet support multiple reports (https://github.com/jenkinsci/cobertura-plugin/issues/82) I think the behaviour was that it would just display the last report generated. So even if we run coverage multiple times, we can only display one 😞